maestro-test/src/main/kotlin/maestro/test/drivers/FakeDriver.kt

```diff
@@ -76,6 +77,12 @@ class FakeDriver : Driver {
         events.add(Event.LaunchApp(appId))
     }
 
+    override fun stopApp(appId: String) {
```

Comment:
Would be great to have an integration test that goes through init flow

Reply:
Actually the test doesn't really fit into our current integration tests because the responsibility of running of the init commands are left up to the caller. The thing that needs to be tested is whether the init commands are parsed properly which is covered in existing tests. Running the init commands from there is covered by existing integration tests.